### PR TITLE
Component initiated book store react project

### PR DESCRIPTION
**Pull request by juaryimami**
### In this project :- 
---
- I tried to lay foundations for your Bookstore website.
-  I created the React part of the app.
- I also set up routing using React
----
#### thank you for reviewing my project!!!!.

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hello @juaryimami 
# Required Changes ♻️

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/components/categories/MainCategory.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const MainCategory = () => (
+  <div>
+    Under construction
+  </div>
+);
+
+export default MainCategory;
```
Comment (andersonlebon) [inline src/components/categories/MainCategory.js]: - [ ] Everything looks fine, but as per the [requirements ](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md#:~:text=Categories%3A,Check%20status%22%20only.), you should display only a button with text "Check status". :+1:

Comment (juaryimami) [reply src/components/categories/MainCategory.js]: ok thank you, I will update it soon

File: src/components/categories/MainCategory.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const MainCategory = () => (
+  <div>
+    Under construction
+  </div>
+);
+
```
Comment (mikethreels) [inline src/components/categories/MainCategory.js]: - [ ] from your response I conclude you understand what the previous CR meant when saying `Everything looks fine, but as per the requirements, you should display only a button with the text "Check status"`, please make the changes before requesting another review.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Juhar, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- linters are passing

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/categories/MainCategory.js (hunk 2 above)
Comment (juaryimami) [reply src/components/categories/MainCategory.js]: you guy's make me confused?

Comment (juaryimami) [reply src/components/categories/MainCategory.js]: you guy's make me confused?

**Review by mikethreels — APPROVED** [review]
Hi Juhar, 

### I would like to make an additional comment, when reviewing your project the previous time these changes weren't pushed yet, please make sure to push your changes before requesting a code review.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — COMMENTED** [review]
Hi @juaryimami, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no updated README file. Please update the new one. [README](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏a

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by sja-thedude — COMMENTED** [review]
## Project Approved 👍 

Hi @juaryimami, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**